### PR TITLE
(PUP-5445) Add Arista support

### PR DESCRIPTION
**Pull request by LuvCurves**

### CODE REVIEW

File: acceptance/config/aio/options.rb [hunk 1]
```diff
@@ -3,12 +3,12 @@
   :is_puppetserver => true,
   :puppetservice => 'puppetserver',
   :'puppetserver-confdir' => '/etc/puppetlabs/puppetserver/conf.d',
-  :restart_when_done => false,
```
Comment (joshcooper) [inline acceptance/config/aio/options.rb]: Please don't remove this.

Comment (LuvCurves) [inline acceptance/config/aio/options.rb]: Sorry, editing error.  Fixed.

File: acceptance/lib/puppet/acceptance/install_utils.rb [hunk 2]
```diff
@@ -146,9 +147,9 @@ def install_repos_on(host, project, sha, repo_configs_dir)
           version = $2
           arch = $3
 
-          if variant =~ /cumulus/ then
-            version = variant
-          end
+	  if variant =~ /cumulus/ then
```
Comment (MikaelSmith) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: This doesn't match indentation. It'd be nice to fix up.

Comment (LuvCurves) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: Fixed

**Comment by MikaelSmith** [comment]
Will merge once it's green.

**Comment by MikaelSmith** [comment]
Retarget of #4384

**Pull request merged**